test/templates.js

```diff
@@ -182,9 +182,6 @@ describe('Templates', function() {
   it('should render fragments containing async components correctly', async () => {
     const req = apos.task.getReq();
     const result = await apos.modules['fragment-page'].renderPage(req, 'page');
-    if (result.match(/error/)) {
```

Comment:
Why remove all these?

Reply:
Now that we are pushing phrases to the browser, there are far too many with the word "error" in them present as data for a simplistic test like this to work. But the test is also unnecessary because none of the expected outputs will be present if the page is a 500 or 404 error.

Reply:
(See the rest of the asserts — they are sufficient.)